### PR TITLE
hook up complete & delete in response view :)

### PR DESCRIPTION
**Pull request by rosasck**
I added the nav bar and fixed the boostrap as well

### CODE REVIEW

File: simcon_project/users/views/view_response.py [hunk 1]
```diff
@@ -12,9 +12,14 @@
...
-
+    if request.method == "POST":
+        response_table = TemplateResponse.objects.all()
+        items_to_delete = request.POST.getlist('delete_items')
+        response_table.filter(pk__in=items_to_delete).delete()
+        return HttpResponseRedirect(reverse('researcher-view'))
     nodes = []
-    num_nodes = TemplateNodeResponse.objects.filter(parent_template_response=response).count()
+    num_nodes = TemplateNodeResponse.objects.filter(
+        parent_template_response=response).count()
```
Comment (devin-w-hill) [inline simcon_project/users/views/view_response.py:22]: This isn't your fault but line 55 throws an error when you try to submit a new overall feedback. Seems to work if it's changed to this:
return HttpResponseRedirect(reverse('view-response', kwargs={'pk': pk}))

File: simcon_project/users/views/view_response.py [hunk 2]
```diff
@@ -12,9 +12,14 @@
 @user_passes_test(is_authenticated)
 def view_response(request, pk):
     response = get_object_or_404(TemplateResponse, pk=pk)
-
+    if request.method == "POST":
+        response_table = TemplateResponse.objects.all()
+        items_to_delete = request.POST.getlist('delete_items')
+        response_table.filter(pk__in=items_to_delete).delete()
+        return HttpResponseRedirect(reverse('researcher-view'))
```
Comment (BroughtMyPopcorn) [inline simcon_project/users/views/view_response.py:19]: I totally should have mentioned this before but there's a few other places on the site where we've used delete buttons that trigger modals to confirm deletion. Seems like we should try to be consistent about it. The settings page for the admin has a good example of what I mean. I can help implement if you'd like

**Comment by devin-w-hill** [comment]
Works as it should, nice. I think we should spruce up the delete buttons to use modals in the future though.

File: simcon_project/users/views/view_response.py (hunk 1 above)
Comment (rosasck) [reply simcon_project/users/views/view_response.py:22]: ill fix this once im off of work! im confused because i didnt touch that...

**Comment by rosasck** [comment]
DONT LOOK AT IT ANYMORE..

**Pull request closed without merging**